docs/reference/indices/index-templates.asciidoc

```diff
@@ -1,18 +1,12 @@
 [[indices-templates]]
```

Comment:
Should we update either the ID or filename so they better match?

Reply:
I'll fix it and add a redirect.